docs/tasks/federation/set-up-coredns-provider-federation.md

```diff
@@ -97,16 +97,22 @@ coredns-provider.conf has below format:
     [Global]
     etcd-endpoints = http://etcd-cluster.my-namespace:2379
     zones = example.com.
+    coredns-endpoints = <coredns-server-ip>:<port>
 
  - `etcd-endpoints` is the endpoint to access etcd.
  - `zones` is the federation domain for which CoreDNS is authoritative and is same as --dns-zone-name flag of `kubefed init`.
+ - `coredns-endpoints` is the endpoint to access coredns server. This is an optional parameter introduced from v1.7 onwards.
 
 *Note: middleware.etcd.zones in CoreDNS configuration and --dns-zone-name
 flag to kubefed init should match.*
 
 
 ## Setup CoreDNS server in nameserver resolv.conf chain
 
+*Note: Following section applies only to versions prior to v1.7 and
```

Comment:
Suggested edit (add "the", "of", and closing * to end italics):

*Note: The following section applies only to versions prior to v1.7 and will be automatically taken care of if the `coredns-endpoints` parameter is configured in `coredns-provider.conf` as described in section above.*